### DISCUSSION TITLE
Check If Record Exists

### DISCUSSION THREAD

**Discussion by cdt-eth** [supabase/supabase#2419] [Questions]
I have a `watchlist` table with an `id` column.

<img width="613" alt="Screen Shot 2021-07-17 at 7 17 23 AM" src="https://user-images.githubusercontent.com/26611339/126035166-cb49155e-efa5-48a9-b2b0-81ae0a90f3b8.png">

I want to check if the movie page im on's `id` matches an `id` in said table column. When I'm on a movie page I have access to the specific `id`. I've looked in the docs for `.is()`, `.contains()`, `.match()`, `.eq()` and I'm not sure which is best for this. 

I'm looking for a boolean value to be returned. 

<img width="949" alt="126035219-d3588c0e-07bc-47d1-9e46-aed7c7731477" src="https://user-images.githubusercontent.com/26611339/126035263-f5ce50d1-2a1c-425f-8f09-dd6d6a2bcbda.png">

**Comment by burggraf** [comment]
You can simply use .eq() and check for the length of the result.

```
 const { data, error} = await supabase
    .from('watchlist')
    .select()
    .eq('id', the_id_variable_I_want_to_check);
if (data?.length > 0) console.log('it exists') else console.log('nope')
```

**Reply by cdt-eth** [reply]
thanks!
